meeteval/io/seglst.py

```diff
@@ -43,13 +53,66 @@ class SegLST(BaseABC):
     # Caches
     _unique = None
 
+    @classmethod
+    def load(cls, file: [Path, str, io.TextIOBase, tuple, list], parse_float=decimal.Decimal) -> 'Self':
+        from meeteval.io.base import _open
+        files = file if isinstance(file, (tuple, list)) else [file]
+
+        parsed = []
+        for f in files:
+            with _open(f, 'r') as fd:
+                parsed.append(cls.parse(fd.read(), parse_float=parse_float))
+
+        return cls.merge(*parsed)
+
+    @classmethod
+    def parse(cls, s: str, parse_float=decimal.Decimal) -> 'Self':
+        """
+        Parses a SegLST from a string.
+
+        >>> SegLST.parse('[{"words": "a b c", "segment_index": 0, "speaker": 0}]')
+        SegLST(segments=[{'words': 'a b c', 'segment_index': 0, 'speaker': 0}])
+        """
+        import simplejson
+
+        def fix_floats(s):
+            """Convert common float keys to decimal"""
+            for k in ('start_time', 'end_time', 'confidence'):
```

Comment:
How about the following idea:
Only change keys, that may be used in meeteval?
I would only convert start_time and end_time to float/decimal, because some Code may do strange things with strings.

Reply:
Done